multicluster/config/samples/clusterset_init/multicluster_cluseterset_template.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: multicluster.crd.antrea.io/v1alpha1
```

Comment:
The file name is incorrect
s/cluseterset/clusterset/

Reply:
done。